Tasks/PipAuthenticateV1/make.json

```diff
@@ -0,0 +1,28 @@
+{
+    "common": [
+        {
+            "module": "../Common/utility-common",
```

Comment:
can you switch to using utility-common-v2 instead (which uses azure-pipelines-task-lib instead of vsts-task-lib)